srv/SpawnEntity.srv

```diff
@@ -0,0 +1,19 @@
+# Spawn an entity (a robot, other object) by name or URI
+
+string name                         # Suggest a name to give to the spawned entity. If left empty, it will be generated.
```

Comment:
SDF and URDF files have a `name` attribute, so should that be used if the `name` field is empty?